### PR TITLE
Update language model to a trie-based LM created from the LibriSpeech LM corpus

### PR DESCRIPTION
**Pull request by reuben**
This supersedes the quick fix that is currently in the tree.

### CODE REVIEW

File: data/lm/README.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+lm.binary was generated from the LibriSpeech normalized LM training text, available [here](http://www.openslr.org/11), following this recipe (Jupyter notebook code):
```
Comment (kdavis-mozilla) [inline data/lm/README.md]: Should we format this a bit better?

**Comment by kdavis-mozilla** [comment]
Looking back at the old commit da760bd it appears as if `vocab.pruned.lm` and `vocab.trie` are the only things which were missed.

As the text is documented as being "Public domain", I don't think the `Attribution.txt` and `License.txt` are needed in this case in contrast to da760bd where the text was CC BY 4.0.

File: data/lm/README.md (hunk 1 above)
Comment (reuben) [reply data/lm/README.md]: How?

Comment (kdavis-mozilla) [reply data/lm/README.md]: Was just thinking at least title, headings, and the like.

**Comment by reuben** [comment]
Updated with the missing files.

**Pull request merged**